src/lib/components/evm-to-cell/evm-to-cell-create.tsx

```
@@ -0,0 +1,99 @@
+import { ExplorerLink } from "../ExplorerLink";
+import { CustomIcon } from "../icon";
+
+interface EvmToCellCreateProps {
+  address: HexAddr20;
+  evmTxHash?: string;
+  isCompact?: boolean;
+}
+
+const EvmToCellCreateContracts = ({ contracts }: { contracts: HexAddr20[] }) =>
+  contracts.map((contract) => (
+    <Flex gap={1} align="center">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)